source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-search/index.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import QueryAssist from 'react-query-assist';
+import { compose } from 'recompose';
+
+import { query, defaultOptions } from '@data';
+import { IFilter } from '@data/containers/with-filtering';
+import { TYPE_NAME as ORGANIZATION } from '@data/models/organization';
+import { TYPE_NAME as GROUP } from '@data/models/group';
+
+// For use after MVP and resolution of this issue:
+// https://github.com/timberio/react-query-assist/issues/25
+// import Footer from './footer';
+
+const inputProps = {
```

Comment:
the query assist component forces us to use emotion which is css-in-js. 
https://github.com/timberio/react-query-assist/blob/master/package.json#L66

Reply:
so, I guess if we have component-specific things that we don't want to write css for, we can use this now.